tribe-customizations/v2/week/grid-body/events-day.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * View: Week View - Events Day
+ *
+ * This is a template override of the file at:
+ * events-calendar-pro/src/views/v2/week/grid-body/events-day.php
+ *
+ * This template override is needed to make the The Events Calendar Pro Extension: Limit Week View Time Range
+ * work with the updated (V2) designs.
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version 5.0.0
+ *
+ * @var WP_Post[] $events The day events post objects.
+ *
+ * @see tribe_get_event() for the additional properties added to the event post object.
+ */
+?>
+
+<div class="tribe-events-pro-week-grid__events-day" role="gridcell">
+	<?php
+	if ( $count == 0 && $ext_options['show_grid'] ) {
+		for ( $i = 0; $i <= $ext_options['grid_end_time'] - $ext_options['grid_start_time']; $i++ ) {
```

Comment:
is this End Time minus Start Time? Is there a reason it's not done with date comparison functions, such as to account for crossing over DST or comparing different time zones?

Reply:
The reason is that these are simple integers (0 to 24) used in the CSS classes, not real date or time.
DST does not influence this at all.
This specific piece will show a grid on the week view. Horizontal dashed lines across the 7 days at every hour. Kind of the extension of the hour tick mark on the left side.
If we have set up a time range, then we need to know how many hours we are showing, in order to draw the same amount of horizontal lines.